cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -41,9 +46,11 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         |** <<administration-security-administration-database-constraints, The `CONSTRAINT MANAGEMENT` privileges>>
         |** <<administration-security-administration-database-tokens, The `NAME MANAGEMENT` privileges>>
         |** <<administration-security-administration-database-all, Granting all database administration privileges>>
+        |** <<administration-security-administration-database-transaction, Granting `TRANSACTION MANAGEMENT` privileges>>
         |* <<administration-security-administration-dbms-privileges, DBMS administration>>
         |** <<administration-security-administration-dbms-custom, Using a custom role to manage DBMS privileges>>
         |** <<administration-security-administration-dbms-privileges-role-management, The dbms `ROLE MANAGEMENT` privileges>>
+        |** <<administration-security-administration-dbms-privileges-user-management, The dbms `USER MANAGEMENT` privileges>>
```

Comment:
This page has inconsistent use of `dbms` in lower case, and `DBMS` - maybe something to fix in another PR?

Reply:
Might be a good idea at some point yes :)